.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+
+<!-- Please fill out any relevant sections and remove those that don't apply -->
+## Motivation
```

Comment:
Is it pertinent, since we are only accepting internal contribution and not expecting external reviewer, the information is already available in Jira.

Reply:
As discussed in vc, we will remove this.